### PR TITLE
{devel}[goolf/1.5.14] Qt 4.8.6 + deps (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
depends on ~~#1317~~ for Python 2.7.9 build dep

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 4 (4 easyconfigs in this PR)
Linux SL 6.5, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/86b3dbd94194f44e14d2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux SL 6.5, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/f57a8551085922f835bf for a full test report.

**Comment by boegel** [comment]
@wpoely86: please review?

File: easybuild/easyconfigs/g/gettext/gettext-0.19.2-goolf-1.5.14.eb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+easyblock = 'ConfigureMake'
+
+name = 'gettext'
+version = '0.19.2'
+
+homepage = 'http://www.gnu.org/software/gettext/'
+description = """GNU `gettext' is an important step for the GNU Translation Project, as it is an asset on which we may
+build many other steps. This package offers to programmers, translators, and even users, a well integrated set of tools
+and documentation"""
+toolchain = {'name': 'goolf', 'version': '1.5.14'} # tweaked by EasyBuild (was: {'name': 'intel', 'version': '2014b'})
```
Comment (wpoely86) [inline easybuild/easyconfigs/g/gettext/gettext-0.19.2-goolf-1.5.14.eb]: add newline before and drop the comment

File: easybuild/easyconfigs/l/libffi/libffi-3.1-goolf-1.5.14.eb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'libffi'
+version = '3.1'
+
+homepage = 'http://sourceware.org/libffi/'
+description = """The libffi library provides a portable, high level programming interface to various calling
+conventions. This allows a programmer to call any function specified by a call interface description at run-time."""
+toolchain = {'name': 'goolf', 'version': '1.5.14'} # tweaked by EasyBuild (was: {'name': 'intel', 'version': '2014b'})
```
Comment (wpoely86) [inline easybuild/easyconfigs/l/libffi/libffi-3.1-goolf-1.5.14.eb]: same here

**Comment by wpoely86** [comment]
looks fine

**Comment by boegel** [comment]
going in, thanks for the review @wpoely86!

**Pull request merged**